pkg/v1/providers/infrastructure-aws/v1.2.0/yttcc/overlay.yaml

```diff
@@ -55,6 +55,7 @@ spec:
         annotations:
           #! VVV TODO(vui) os-name handling
           run.tanzu.vmware.com/resolve-os-image: #@ "ami-region={},os-name=ubuntu".format(data.values.AWS_REGION)
+          tkg.tanzu.vmware.com/test-annotation: test
```

Comment:
what is this used for?

Reply:
please see the PR description. This is only included for demo purpose to show the resultant diffs, and will be removed before merging.